### PR TITLE
Test - mapa - color distinto de punto en imagen si pones pista de ocultar

### PR DESCRIPTION
**Pull request by fermarinsanchez**
fix(QuestionImage): Only show hide marker if user consume hint type "hide-response"

### CODE REVIEW

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```
Comment (MIGUELez11) [inline plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js:1]: The below code was correctly formated...

```suggestion
```

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js [hunk 2]
```diff
@@ -34,29 +35,30 @@ export function QuestionImage({ src, markers, values, clue }) {
       <LeebraryImage className={classes.image} src={src} />
       {markers && markers.list
         ? markers.list.map((marker, index) => {
```
Comment (MIGUELez11) [inline plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js:37]: If you see in detail, there must be 2 identation levels:

```js
markers && markers.list
	? markerts.list.map((marker, index) => {
			// ...
		})
	: null;
```

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js [hunk 3]
```diff
@@ -34,29 +35,30 @@ export function QuestionImage({ src, markers, values, clue }) {
...
-            let { backgroundColor } = markers;
-            if (values) {
-              backgroundColor = index === values[index] ? COLORS.fatic02 : COLORS.fatic01;
-            }
+          let { backgroundColor } = markers;
+          if (values) {
+            backgroundColor = index === values[index] ? COLORS.fatic02 : COLORS.fatic01;
+          }
```
Comment (MIGUELez11) [inline plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js:41]:
```suggestion
	          let { backgroundColor } = markers;
	          if (values) {
	            backgroundColor = index === values[index] ? COLORS.fatic02 : COLORS.fatic01;
	          }
```

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js [hunk 4]
```diff
@@ -34,29 +35,30 @@ export function QuestionImage({ src, markers, values, clue }) {
...
+              style={{
+                top: marker.top,
+                left: marker.left,
+                backgroundColor,
+              }}
+            >
+              {markers.type === 'letter' ? numberToEncodedLetter(index + 1) : index + 1}
+            </Box>
+          );
+        })
```
Comment (MIGUELez11) [inline plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js:61]:
```suggestion
	          return (
	            <Box
	              key={index}
	              className={classes.marker}
	              style={{
	                top: marker.top,
	                left: marker.left,
	                backgroundColor,
	              }}
	            >
	              {markers.type === 'letter' ? numberToEncodedLetter(index + 1) : index + 1}
	            </Box>
	          );
	        })
```

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js [hunk 5]
```diff
@@ -34,29 +35,30 @@ export function QuestionImage({ src, markers, values, clue }) {
...
+            backgroundColor = index === values[index] ? COLORS.fatic02 : COLORS.fatic01;
+          }
 
-            if (clue) {
-              backgroundColor = clue.indexs.includes(index) ? COLORS.fatic03 : backgroundColor;
-            }
+          if (clue) {
+            backgroundColor = clue.indexs.includes(index) ? COLORS.fatic03 : backgroundColor;
+          }
+          if (!markers?.canShowHintMarker && marker.hideOnHelp) return null;
```
Comment (MIGUELez11) [inline plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js:46]:
```suggestion
	          if (clue) {
	            backgroundColor = clue.indexs.includes(index) ? COLORS.fatic03 : backgroundColor;
	          }
	          if (!markers?.canShowHintMarker && marker.hideOnHelp) return null;
```

**Review by MIGUELez11 — CHANGES_REQUESTED** [review]
Please remove disable eslint...
And return to the correct identation.

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js (hunk 1 above)
Comment (fermarinsanchez) [reply plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js]: fixed

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js (hunk 2 above)
Comment (fermarinsanchez) [reply plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js]: fixed

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js (hunk 3 above)
Comment (fermarinsanchez) [reply plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js]: fixed!

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js (hunk 4 above)
Comment (fermarinsanchez) [reply plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js]: fixed

File: plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js (hunk 5 above)
Comment (fermarinsanchez) [reply plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js]: fixed

Comment (fermarinsanchez) [reply plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js]: fixed

**Review by MIGUELez11 — APPROVED** [review]

**Pull request merged**